appcenter/android/src/main/java/com/microsoft/appcenter/reactnative/appcenter/AppCenterReactNativeListener.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.appcenter.reactnative.appcenter;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.modules.core.DeviceEventManagerModule;
+import com.microsoft.appcenter.AuthTokenCallback;
+import com.microsoft.appcenter.AuthTokenListener;
+
+public class AppCenterReactNativeListener implements AuthTokenListener {
+
+    private static final String ON_ACQUIRE_AUTH_TOKEN_EVENT = "OnAcquireAuthTokenEvent";
+
+    private ReactApplicationContext mReactApplicationContext;
+
+    private AuthTokenCallback mCallback;
+
+    private boolean mAuthTokenEventSent = false;
```

Comment:
Setting fields explicitly to null, false or 0 is usually not what we do in our Java/C# code.